products/ansible.md

```diff
@@ -77,8 +76,8 @@ for upcoming release details.
 
 ## Python Compatibility
 
```

Comment:
How about this:

Ansible|Ansible-Core Version Dependency|Controller Python|Target Python
---|---|---|---
3|2.10|2.7,3.5-3.9|2.6-2.7,3.5-3.9
4|2.11|2.7,3.5-3.9|2.6-2.7,3.5-3.9
5|2.12|3.8-3.10|2.6,3.5-.310
6|2.13|3.8-3.10|2.7,3.5-3.10
7|2.14|3.9 - 3.11|2.7, 3.5-3.11

Reply:
(Links each Ansible release to the corresponding `ansible-core` version, and uses the versions from there.

Reply:
Shouldn't we do this on the `ansible` page instead ?

Reply:
The suggestion is on the `ansible` page.

Reply:
I ended up documenting only ansible-core compatibility on the ansible page : https://deploy-preview-2724--endoflife-date.netlify.app/ansible. I also added a link to the ansible-core page for easily get the Python compatibility information.

Displaying information from another page is feasible but not directly in the page / template. Working only in a template will not work due to a Liquid exception : `{{ site.pages | where: "permalink", "/ansible-core" | map: "releases" | where: "releaseCycle", "2.10" | map: "pythonVersionsControlNode" }}` returns `Liquid Exception: no implicit conversion of String into Integer in products/ansible.md`.

Another way to retrieve `pythonVersionsControlNode` and `pythonVersionsManagedNode` versions  would be to use a new custom filter, but that does not worth it if it is only used on the ansible page.

Reply:
Ok, I re-integrated Python compatibility on Ansible page.